components/video-player.tsx

```diff
@@ -41,8 +41,31 @@ type SizedEvent = {
   }
 };
 
-const VideoPlayer: React.FC<Props> = ({ playbackId, poster, onLoaded, onError }) => {
-  const videoRef = useRef<HTMLVideoElement | null>(null);
+export interface HTMLVideoElementWithPlyr extends HTMLVideoElement {
+  plyr: Plyr
+}
+
+const useCombinedRefs = function (...refs: (((instance: HTMLVideoElementWithPlyr | null) => void) | MutableRefObject<HTMLVideoElementWithPlyr | null> | null)[]) {
+  const targetRef = useRef(null);
+
+  useEffect(() => {
+    refs.forEach(ref => {
+      if (!ref) return;
+
+      if (typeof ref === 'function') {
+        ref(targetRef.current);
+      } else {
+        ref.current = targetRef.current;
+      }
+    });
+  }, [refs]);
+
+  return targetRef;
+};
```

Comment:
Yea, I think that makes sense.  I couldn't find a directory where it would make sense to add it already.  How would you feel about putting it in `util/use-combined-refs.ts`?